src/api/middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../../config/logging');
+const jwt = require('jsonwebtoken');
+
+const config = require('../../config/config');
+const { StatusCodes } = require('http-status-codes');
+
+const handle = (req, res, next) => {
+    if (req.headers.authorization) {
+        jwt.verify(req.headers.authorization.split(' ')[1], config.SECRET, (err) => {
+            if (err) {
+                logger.error(`FORBIDDEN: bad token`);
+                res.status(StatusCodes.FORBIDDEN).end('Failed to authenticate token');
+            } else {
+                return next();
+            }
+        });
+    } else {
+        logger.error(`FORBIDDEN: there is no token`);
+        res.status(StatusCodes.FORBIDDEN).end('Not token provided');
```

Comment:
здесь не 403 forbidden, а 401 

In case of the HTTP Authorization header is absent in the request, the middleware should stop further controller method execution and return HTTP 401 code (Unauthorized Error) and standard error message.